tests/integration-tests/tests/schedulers/test_slurm_accounting.py

```diff
@@ -203,55 +216,3 @@ def test_slurm_accounting(
     _test_slurm_accounting_password(remote_command_executor)
     _test_slurm_accounting_database_name(remote_command_executor, custom_database_name)
     _test_that_slurmdbd_is_running(remote_command_executor)
-
-
-@pytest.mark.usefixtures("os", "instance", "scheduler")
-def test_slurm_accounting_disabled_to_enabled_update(
-    region,
-    pcluster_config_reader,
-    database_factory,
-    vpc_stack_for_database,
-    request,
-    test_datadir,
-    test_resources_dir,
-    clusters_factory,
-    scheduler_commands_factory,
-):
-    database_stack_name = database_factory(
-        request.config.getoption("slurm_database_stack_name"),
-        str(test_datadir),
-        region,
-    )
-
-    database_stack_outputs = get_infra_stack_outputs(database_stack_name)
-    public_subnet_id = vpc_stack_for_database.get_public_subnet()
-    private_subnet_id = vpc_stack_for_database.get_private_subnet()
-
-    # First create a cluster without Slurm Accounting enabled
-    cluster_config = pcluster_config_reader(public_subnet_id=public_subnet_id, private_subnet_id=private_subnet_id)
-    cluster = clusters_factory(cluster_config)
-
-    remote_command_executor = RemoteCommandExecutor(cluster)
-
-    _test_that_slurmdbd_is_not_running(remote_command_executor)
-
-    config_params = _get_slurm_database_config_parameters(database_stack_outputs)
-
-    # Then update the cluster to enable Slurm Accounting
-    updated_config_file = pcluster_config_reader(
-        config_file="pcluster.config.update.yaml",
-        public_subnet_id=public_subnet_id,
-        private_subnet_id=private_subnet_id,
-        **config_params,
-    )
-    cluster.update(str(updated_config_file), force_update="true")
-
-    remote_command_executor = RemoteCommandExecutor(cluster)
-    scheduler_commands = scheduler_commands_factory(remote_command_executor)
-
-    # Test for successful Slurm Accounting start up
-    _test_that_slurmdbd_is_running(remote_command_executor)
-    _test_successful_startup_in_log(remote_command_executor)
-    _test_slurmdbd_log_exists_in_log_group(cluster)
-    _test_slurmdb_users(remote_command_executor, scheduler_commands, test_resources_dir)
-    _test_jobs_get_recorded(scheduler_commands)
```

Comment:
Shouldn't we execute these checks once we enable slurm accounting?

Reply:
After the cluster is updated to enable slurm accounting, these checks are run (lines 197-202), previously these checks were run in both test_slurm_accounting and test_slurm_accounting_disabled_to_enabled_update.